app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+
+    command, source_file, new_file = parts[0], parts[1], parts[2]
+
+    if command != "cp" and len(parts) != 3:
```

Comment:
The condition `if command != "cp" and len(parts) != 3:` should use `or` instead of `and`. The current condition will only return if both conditions are true, which is not the intended logic. It should return if either the command is not 'cp' or the length of parts is not 3.